### ISSUE TITLE
Top Stats

### ISSUE THREAD

**Issue by Tigerpanzer02** [Plajer-Lair/Village_Defense#4]
### Sort description of issue
When doing this command I become a Error in Console and it isn´t working ;)

## Error code
```
[22:01:43 INFO]: Tigerpanzer_02 issued server command: /vd top Kills

[22:01:43 INFO]: [Village Debugger] Running task 'Village API getStats(kills) run'

[22:01:43 ERROR]: null

org.bukkit.command.CommandException: Unhandled exception executing command 'vd' in plugin VillageDefense v3.6.2

at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot.jar:git-PaperSpigot-"4c7641d"]

at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:146) ~[spigot.jar:git-PaperSpigot-"4c7641d"]

at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:666) ~[spigot.jar:git-PaperSpigot-"4c7641d"]

at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1189) [spigot.jar:git-PaperSpigot-"4c7641d"]

at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:1001) [spigot.jar:git-PaperSpigot-"4c7641d"]

at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot.jar:git-PaperSpigot-"4c7641d"]

at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot.jar:git-PaperSpigot-"4c7641d"]

at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [spigot.jar:git-PaperSpigot-"4c7641d"]

at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_171]

at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_171]

at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [spigot.jar:git-PaperSpigot-"4c7641d"]

at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:774) [spigot.jar:git-PaperSpigot-"4c7641d"]

at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378) [spigot.jar:git-PaperSpigot-"4c7641d"]

at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:713) [spigot.jar:git-PaperSpigot-"4c7641d"]

at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:616) [spigot.jar:git-PaperSpigot-"4c7641d"]

at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]

Caused by: java.lang.NullPointerException

at java.util.regex.Matcher.appendReplacement(Matcher.java:804) ~[?:1.8.0_171]

at java.util.regex.Matcher.replaceAll(Matcher.java:955) ~[?:1.8.0_171]

at java.lang.String.replaceAll(String.java:2223) ~[?:1.8.0_171]

at pl.plajer.villagedefense3.commands.GameCommands.sendTopStatistics(GameCommands.java:97) ~[?:?]

at pl.plajer.villagedefense3.commands.MainCommand.onCommand(MainCommand.java:238) ~[?:?]

at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot.jar:git-PaperSpigot-"4c7641d"]

... 15 more
```

## Additional info
- Spigot version: 1.8
- Plugin version: 3.6.2
- *Addons with their versions: Not needed

(* optional)

**Comment by Plajer** [comment]
"Fixed" in bb72237e0e95ef714eb57eef3d7c7981f542a034 but issue for Unknown Players won't be fixed for mysql users if occured.

**Comment by Plajer** [comment]
Probably fixed in above commit, need to test it.

Closing for now.
